src/main/java/org/jabref/logic/importer/fetcher/MedlineFetcher.java

```diff
@@ -80,6 +80,8 @@ private static String replaceCommaWithAND(String query) {
             XMLInputFactory inputFactory = XMLInputFactory.newFactory();
             XMLStreamReader streamReader = inputFactory.createXMLStreamReader(ncbi.openStream());
 
+            System.out.println("parsing xml");
```

Comment:
Why can't you use the logging infrastructure and set the log level accordingly?

Include that in `log4j2.xml`:

```
        <Logger name="org.jabref.logic.importer.fetcher" level="DEBUG">
            <AppenderRef ref="CONSOLE"/>
        </Logger>
```

Reply:
@koppor  because of  #5475 and it took me a while to figure out how to pass arguments to the jpackage started application

Reply:
The problem is that the fetcher fails only in the jpackage application and works flawlessly when running from gradle